app/config/FrontendAppConfig.scala

```diff
@@ -17,7 +17,7 @@
 package config
 
 import com.google.inject.{Inject, Singleton}
-import models.{C79Certificate, DutyDefermentStatement, FileRole, PostponedVATStatement, SecurityStatement, UserAnswers}
+import models._
```

Comment:
Could you please revert back as its good practice to specify the specific models (that are being used) and using all the models increase the compile time. We should only use ._ when either we are using most of the models form the folder or all of them. Here we are only using only 6 out of lot